diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter2/kdoc/KdocFormatting.kt

```diff
@@ -237,6 +237,19 @@ class KdocFormatting(configRules: List<RulesConfig>) : DiktatRule(
     private fun checkEmptyLineBeforeBasicTags(basicTags: List<KDocTag>) {
         val firstBasicTag = basicTags.firstOrNull() ?: return
 
+        // Unlike other tags, @property and @constructor tags are inside separate KDOC_SECTION,
```

Comment:
Next time my suggestion to split such functional bug fixes like this one with the above refactoring :) It's difficult to review it

Reply:
Agreed. But refactoring is also partly related to this issue, so I decided to combine refactoring and bug fix into one pull request.